x-pack/legacy/plugins/ml/public/data_frame/pages/transform_management/services/transform_service/delete_transform.ts

```diff
@@ -38,3 +38,40 @@ export const deleteTransform = async (d: DataFrameTransformListRow) => {
   }
   refreshTransformList$.next(REFRESH_TRANSFORM_LIST_STATE.REFRESH);
 };
+
+export const bulkDeleteTransforms = async (dataFrames: DataFrameTransformListRow[]) => {
```

Comment:
is `deleteTransform` needed now as `bulkDeleteTransforms` could be called with a list of 1.

Reply:
It is not. Updated all the actions to accept a list in 7988c5d5c1077aaeb9f527ff48928e3019bd692e